scripts/newinstall.sh

```diff
@@ -386,9 +386,12 @@ miniconda::install() {
 	(
 		set -e
 
-		# the miniconda installer seems to complains if the filename does not end
-		# with .sh
-		tmpfile=$(mktemp -t "XXXXXXXX.${miniconda_file_name}")
+		# XXX the miniconda installer seems to complains if the filename does not
```

Comment:
It is emphasis that we're working around what is arguably a bug in the miniconda installer.